src/frontend/js/modules/themes/apply-theme.service.js

```diff
@@ -0,0 +1,41 @@
+const hexToHsl = require('hex-to-hsl');
+
+angular.module('esn.themes').factory('applyThemeService', applyThemeService);
+
+// TODO (esn-frontend-common-libs#51): Write tests for this module
+function applyThemeService() {
+  const camelToKebabCase = text => text.replace(/[A-Z]/g, letter => `-${letter.toLowerCase()}`);
```

Comment:
I'm not sure if our frontend code supports arrow function. Please check again with @tuanlc .
As I checked in webpack config, we don't have babel yet

Reply:
I am on it, we will have it today